<ui version="4.0" >
 <class>QuickSearchDialog</class>
 <widget class="QDialog" name="QuickSearchDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Quick Tag Search</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Tag:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="tagCombo" >
     <item>
      <property name="text" >
       <string>Game number</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>White</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Black</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Event</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Site</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Round</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Date</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Result</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>ECO</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Length</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Value:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="valueEdit" />
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>&amp;Find</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tagCombo</tabstop>
  <tabstop>valueEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
